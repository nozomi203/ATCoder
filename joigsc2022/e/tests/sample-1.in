7 100
20
80
50
40
20
80
10
