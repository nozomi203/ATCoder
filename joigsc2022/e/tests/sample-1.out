4 10
4 10
3 10
2 90
2 10
1 90
1 10
